core/modules/xrdsvc/SsiRequest.cc

```diff
@@ -211,4 +210,63 @@ void SsiRequest::Finished(XrdSsiRequest& req, XrdSsiRespInfo const& rinfo, bool
     LOGS(_log, LOG_LVL_DEBUG, "RequestFinished " << type);
 }
 
+bool SsiRequest::reply(char const* buf, int bufLen) {
+    Status s = SetResponse(buf, bufLen);
+    if (s != XrdSsiResponder::wasPosted) {
+        LOGS(_log, LOG_LVL_ERROR, "DANGER: Couldn't post response of length=" << bufLen);
+        return false;
+    }
+    return true;
+}
+
+bool SsiRequest::replyError(std::string const& msg, int code) {
+    Status s = SetErrResponse(msg.c_str(), code);
+    if (s != XrdSsiResponder::wasPosted) {
+        LOGS(_log, LOG_LVL_ERROR, "DANGER: Couldn't post error response " << msg);
+        return false;
+    }
+    return true;
+}
+
+bool SsiRequest::replyFile(int fd, long long fSize) {
+    util::Timer t;
+    t.start();
+    Status s = SetResponse(fSize, fd);
```

Comment:
"Status status" would be better than "Status s". Oddly, I don't care about "Timer t". I guess, since t isn't logically important to the function.